### DISCUSSION TITLE
171

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#134] [General]
# 潮流周刊第171期 - 看虞美人

封面图来源于假期径山花海拍摄的虞美人，看牌子上写的是野罂粟，貌似是第一次认真观察这种花，远看很漂亮，特别是有风的时候这个效果太好了。

https://weekly.tw93.fun/posts/171-%E7%9C%8B%E8%99%9E%E7%BE%8E%E4%BA%BA

**Comment by arlendp** [comment]
发布日期写错了哦

**Reply by tw93** [reply]
我去，谢谢提醒，我还活在2023

**Comment by luo0412** [comment]
😂是99块钱洗5件

**Reply by tw93** [reply]
好像有时候有，哈哈哈，我感觉108也是很够的

**Comment by Orange-Yellow** [comment]
图片如果能点击放大预览就更好了

**Reply by tw93** [reply]
哈哈哈 那就安排

**Reply by Orange-Yellow** [reply]
我为啥还可以收到你回复其他人的邮件啊？

**Reply by tw93** [reply]
是不是你在其他地方订阅了导致，我这边没有收到过，比如说你的github通知里面设置了邮件通知就有可能

此外图像点击放大pc支持了，你可以点击看看，应该符合使用

**Reply by Orange-Yellow** [reply]
赞！！！但是好像封面图没法放大呢~
没有单独订阅什么呢？图片就是我收到了其他人的回复
![1712707709093.png](https://pic.ziyuan.wang/user/guest/2024/04/1712707709093_b5e90483bf156.png)

**Reply by Orange-Yellow** [reply]
这期封面是视频，所以没法放大 ^_^

**Reply by tw93** [reply]
正解

**Reply by tw93** [reply]
https://github.com/settings/notifications
去这个页面设置即可
<img width="1229" alt="image" src="https://github.com/tw93/weekly/assets/8736212/47a62854-2669-4ef0-93cb-8b874a81dd51">

**Comment by luo0412** [comment]
您好：我已收到您的邮件，我将尽快查收处理。祝您生活愉快

**Comment by marsbaiyun** [comment]
请问生活小技巧那个截图，是在mac上运行app截的图，还是app截的多张图在pc拼接的啊？

**Reply by tw93** [reply]
https://shottr.cc/
试试这个，我一般是直接截图
